packages/sds/src/message_channel/message_channel.ts

```diff
@@ -38,13 +37,31 @@ export class MessageChannel extends TypedEventEmitter<MessageChannelEvents> {
   private acknowledgements: Map<string, number>;
   private incomingBuffer: Message[];
   private localHistory: { timestamp: number; historyEntry: HistoryEntry }[];
-  private channelId: ChannelId;
+  public channelId: ChannelId;
```

Comment:
nit: if it is `readonly`

```suggestion
  public readonly channelId: ChannelId;
```